app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6

```diff
@@ -79,6 +80,27 @@ Object.assign(MediaElementPlayer.prototype, {
    * @type {Object}
    */
   playlistItemsObj: {
+    /**
+     * Add click listeners forplaylist item links
+     * @function addRelatedItemListeners
+     * @return {void}
+     */
+    addRelatedItemListeners() {
+      // Handle click on entire RelatedItems area
+      // Filter only <a> element clicks; disregard all others
+      if (this.$relatedItems) {
```

Comment:
If `$relatedItems` is a jQuery variable reference (preceding a variable title with `$` is a common, good pattern to follow), then you might need to check `if (this.$relatedItems.length > 0)` as an empty jQuery variable reference will return `true`.